src/app/modules/ix-forms/components/ix-button-group/ix-button-group.component.spec.ts

```diff
@@ -0,0 +1,58 @@
+import { NgControl } from '@angular/forms';
+import { MatButtonToggleChange, MatButtonToggleModule } from '@angular/material/button-toggle';
+import { Spectator, createComponentFactory } from '@ngneat/spectator/jest';
+import { MockComponent } from 'ng-mocks';
+import { IxErrorsComponent } from 'app/modules/ix-forms/components/ix-errors/ix-errors.component';
+import { IxButtonGroupComponent } from './ix-button-group.component';
+
+describe('IxButtonGroupComponent', () => {
+  let spectator: Spectator<IxButtonGroupComponent>;
+  const createComponent = createComponentFactory({
+    component: IxButtonGroupComponent,
+    imports: [MatButtonToggleModule],
+    declarations: [MockComponent(IxErrorsComponent)],
+    providers: [NgControl],
+  });
+
+  beforeEach(() => (spectator = createComponent()));
+
+  it('should create', () => {
+    expect(spectator.component).toBeTruthy();
+  });
+
+  it('should disable button group', () => {
+    spectator.component.setDisabledState(true);
+    expect(spectator.component.isDisabled).toBe(true);
+  });
+
+  it('should emit value change', () => {
+    const onChangeSpy = jest.spyOn(spectator.component, 'onChange');
+    spectator.component.onValueChanged({ value: 'test' } as MatButtonToggleChange);
+    expect(onChangeSpy).toHaveBeenCalledWith('test');
+  });
+
+  describe('setDisabledState()', () => {
```

Comment:
I think ideally you wouldn't be calling these methods. These could be set to `protected`.

A better approach is to create a `FormControl` to interact with (set value, read value, set disabled) and check how it communicates with mat-button-toggle.
For example, see `IxInputComponent`.